recommend/rcmd.go

```diff
@@ -60,6 +61,7 @@ type UserFeaturer interface {
 }
 
 type ItemFeaturer interface {
+	GetItemsFeature(context.Context, []int) ([]Tensor, error)
 	GetItemFeature(context.Context, int) (Tensor, error)
```

Comment:
@nmreadelf you can delete the `GetItemFeature` interface and its dependence and implementation